src/desktop/components/article/client/view.coffee

```diff
@@ -42,8 +42,6 @@ module.exports = class ArticleView extends Backbone.View
     @setupFollowButtons()
     @setupImageSets()
     @resetImageSetPreview()
-    if @article.attributes.channel?.id == sd.GALLERY_INSIGHTS_CHANNEL
-      @setupMarketoStyles()
```

Comment:
I see some other references to `GALLERY_INSIGHTS_CHANNEL` in the codebase. Can those be removed as well?

Reply:
This var is still used to redirect GI insights articles to `partners.artsy.net`